### PR TITLE
Add Gabriel's part to AboutUs

### LINKED ISSUE #18: Update 'About Us' for Gabriel Goh
Update the about pages to include information regarding Gabriel Goh

### PR DESCRIPTION
**Pull request by kawaiigabrielneko**
Resolves #18

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+<img src="https://upload.wikimedia.org/wikipedia/en/b/b1/Portrait_placeholder.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/kawaiigabrielneko)]
+[[portfolio](team/kawaiigabrielneko.md)]
 
 * Role: Developer
-* Responsibilities: UI
+* Responsibilities: Insurance, Glossary
```
Comment (xianlinc) [inline docs/AboutUs.md:59]: Minor standardisation issue.  
Can you change the comma (,) to a plus (+) ?
e.g Insurance + Glossary

File: docs/AboutUs.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Gabriel Goh Kheng Lin
 
-<img src="images/johndoe.png" width="200px">
+<img src="https://upload.wikimedia.org/wikipedia/en/b/b1/Portrait_placeholder.png" width="200px">
```
Comment (xianlinc) [inline docs/AboutUs.md:53]: I think that the CS2103 grader will be checking for your picture upload, so you can't use a url.
Could you upload a png of yourself to the images folder, so the grader can pick it up.
Use this format 'images/githbub_username_in_lower_case.png'

**Review by xianlinc — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 2 above)
Comment (JoelChanZhiYang) [reply docs/AboutUs.md:53]: I think this is allowed

Comment (rldefa) [reply docs/AboutUs.md:53]: but why tho

Comment (xianlinc) [reply docs/AboutUs.md:53]: Oh i see what you are referring to. In order for the grader to still pick it up, Gabriel may have to upload a placeholder image named his github username, if he wants to use a placeholder image. Not sure if url is allowed

**Comment by kawaiigabrielneko** [comment]
I have made the required changes

**Review by JoelChanZhiYang — APPROVED** [review]
LGTM

**Pull request merged**